docs/hpc/03_storage/06_data_management.mdx

```diff
@@ -0,0 +1,115 @@
+
+## Data Storage options in the HPC Environment
+### User Home Directories
+Every individual user has a home directory (under **`/home/$USER`**, environment variable **`$HOME`**) for permanently storing code and important configuration files. Home Directories provide limited storage space (**50 GB**) and inodes (files) **30,000** per user. Users can check their quota utilization using the [myquota](http://www.info-ren.org/projects/ckp/tech/software/version/myquota.html) command.
+
+User home directories are backed up daily and old files under **`$HOME`** are not purged.
+
+The User home directories are available on all HPC clusters (Greene) and on every cluster node (login nodes, compute nodes) as well as and Data Transfer Node (gDTN). 
+
+:::note
+-   **Avoid changing file and directory permissions in your home directory to allow other users to access files.** User Home Directories are not ideal for sharing files and folders with other users. HPC Scratch of Research Project Space (RPS) are better file systems for sharing data.
+-   **One of the common issues that users report regarding their home directories is running out of inodes,** i.e. the number of files  stored under their home exceeds the inode limit, which by default is set to 30,000 files. This typically occurs when users install software under their home directories, for example, when working with Conda and Julia environments, that involve many small files. 
+-   To find out the current space and inode quota utilization and the distribution of files under your home directory, please see: [Understanding user quota limits and the myquota command.](./07_best_practices.md#user-quota-limits-and-the-myquota-command) 
+-   **Working with Conda environments:** To avoid running out of inode limits in home directories, the HPC team recommends **setting up conda environments with Singularity overlay images**
+:::
+
+### HPC Scratch
+The HPC scratch file system is the HPC file system where most of the users store research data needed during the analysis phase of their research projects. The scratch file system provides ***temporary*** storage for datasets needed for running jobs. 
+
+Files stored in the HPC scratch file system are subject to the <ins>**HPC Scratch old file purging policy:** Files on the /scratch file system that have not been accessed for 60 or more days will be purged.</ins>
+
+Every user has a dedicated scratch directory (**/scratch/$USER**) with **5 TB** disk quota and **1,000,000 inodes** (files) limit per user. 
+
+The scratch file system is available on all nodes (compute, login, etc.) on Greene as well as Data Transfer Node (gDTN).
+
+:::note
+-   There are ***No Back ups*** of the scratch file system. ***Files that were deleted accidentally or removed due to storage system failures CAN NOT be recovered.***
```

Comment:
It is possible to add nested-admonitions to create emphasis : https://docusaurus.io/docs/markdown-features/admonitions#nested-admonitions

Maybe convert the first one to a danger?

Reply:
I had the same problem with with nested admonitions in the list.  I moved the first bullet point to it's own warning and then the other three bullets about what to do since there are no backups seem to go well together in a list inside the note.  I just pushed this change.  How does it look?

Reply:
Yeah, it took me a while to figure out how to do it. Here's an example: https://github.com/NYU-ITS/rts-docs-dev/blob/c7e6fc5a1910aee44e1275f45628150125f46c54/docs/srde/02_user_guide/01_connecting.mdx?plain=1#L14-L24

Reply:
Thanks so much.  I went with a slightly different style as mentioned above.  I'll resolve this ticket, but please let me know if you'd rather I change it in a new comment.